src/viewProviders/schemas.ts

```diff
@@ -131,14 +134,44 @@ export class SchemasViewProvider implements vscode.TreeDataProvider<SchemasViewP
   }
 
   /** Convenience method to revert this view to its original state. */
-  reset(): void {
-    setContextValue(ContextValues.schemaRegistrySelected, false);
-    this.environment = null;
-    this.schemaRegistry = null;
-    this.treeView.description = "";
```

Comment:
Looks like we may need to update the `description` in this new `setSchemaRegistry` since we can get into a mixed empty state, one example:
- When a direct connection SR is focused but that connection is deleted/disconnected: 
![image](https://github.com/user-attachments/assets/af3fdc59-2251-4e49-9476-176ddc0490aa)

Reply:
Fixed, thanks! Had overlapping events firing, and one short circuited properly, but the first one took longer and ended up losing / winning the race and filling in with half-bad data.